Mobile/ShopLens/ShopLens/Extensions/ExtensionMethods.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ShopLens.Extensions
+{
+    static class ExtensionMethods
+    {
+        public static string FirstCharToUpper(this string input)
+        {
+            switch (input)
+            {
+                case null:
+                    throw new ArgumentNullException(nameof(input));
+
+                case "":
+                    throw new ArgumentException($"{nameof(input)} cannot be empty", nameof(input));
```

Comment:
Don't think this should throw if you ask me, just return empty string.

Reply:
Well, I didn't write this method, but I thought I should leave the throws.